### PR TITLE
Configure logging

### PR DESCRIPTION
**Pull request by amaalali**

### CODE REVIEW

File: test/resources/logback-test.xml [hunk 1]
```diff
@@ -6,6 +6,8 @@
         </encoder>
     </appender>
 
+    <logger name="application" level="OFF"/>
```
Comment (MatthewAlexanderBeck) [inline test/resources/logback-test.xml:9]: This is just for turning off the unit test logging noise?

File: app/logging/Logging.scala [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ * limitations under the License.
+ */
+
+package logging
+
+import play.api.Logger
+
+trait Logging {
+
+  protected val logger: Logger = Logger(s"application.${this.getClass.getCanonicalName}")
```
Comment (MatthewAlexanderBeck) [inline app/logging/Logging.scala:23]: Being a val how does this change for each class? Would you mind quickly going over this with me.

File: test/resources/logback-test.xml (hunk 1 above)
Comment (amaalali) [reply test/resources/logback-test.xml:9]: Yup, that exactly

File: app/logging/Logging.scala (hunk 2 above)
Comment (TheNickWilson) [reply app/logging/Logging.scala:23]: This is on a trait that various classes extend.  The val gets initialised on the class itself, not on the trait - so `this.getClass` is actually being called on e.g. `ReferenceDataListController` etc.

**Review by MatthewAlexanderBeck — APPROVED** [review]

**Pull request merged**